build/external-dependencies.props

```diff
@@ -384,7 +384,7 @@
 
   <ItemGroup Condition="'$(DotNetAdditionalRestoreSources)'!=''">
     <ExternalDependency
-      Update="@(ExternalDependency->WithMetadataValue('Source', '$(DotNetCoreFeed)'));@(ExternalDependency->WithMetadataValue('Source', '$(RoslynFeed)'))"
+      Update="@(ExternalDependency)"
```

Comment:
Out of curiosity, what packages did this miss?

Reply:
Packages that we are currently pulling from nuget.org, like Microsoft.Build.Framework. This one is now building in PCP